guides/20240904_guide_setting_up_daytona_aws_provider.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Setting Up Daytona AWS Provider"
+description: "A step-by-step guide to setting up and configuring the Daytona AWS Provider for managing workspace projects on Amazon EC2 instances."
+date: 2024-09-04
+author: "Kiran Naragund"
+tags: ["aws", "ec2", "daytona"]
+---
+
+# Setting Up Daytona AWS Provider
+
+## Introduction
+
+The AWS Provider allows Daytona to create and manage workspace projects on Amazon EC2 instances. The provider integrates Daytona with AWS, enabling you to manage workspaces in a scalable and flexible cloud environment. To use the AWS Provider, you need an AWS programmatic access user with AmazonEC2FullAccess permissions. This policy grants the necessary permissions to manage EC2 instances, which is crucial for Daytona’s workspace project creation and management.
+
+In this guide, we will walk you through the setup and configuration of the Daytona AWS Provider, as well as creating an IAM user with the necessary permissions.
+
+### Prerequisites
+
+To follow along with this guide, you should have:
+- A basic understanding of [AWS](../definitions/20240904_definition_aws.md), [AWS IAM](../definitions/20240904_definition_aws_iam.md) and [EC2 Instances](../definitions/20240904_definition_ec2_instances.md). 
+- Access to an AWS account with programmatic access and appropriate permissions.
+- installed Daytona [install from [here](https://www.daytona.io/docs/installation/installation/)]
```

Comment:
add note for the version of daytona used to test this

Reply:
please update to the latest version and change the relevant bits of this article as some stuff is outdated